core/tests/linter_tests/invalid_ddescribe_fdescribe.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview invalid syntax .ts file.
```

Comment:
Start with `I` (caps), also include more info like for whcih kinda test this file is used (here and in all other similar files)

Reply:
Done.